src/quo2/components/dropdowns/dropdown/properties.cljs

```diff
@@ -0,0 +1,130 @@
+(ns quo2.components.dropdowns.dropdown.properties
+  (:require [quo2.foundations.colors :as colors]))
+
+(def backgrounds #{:photo :blur})
+
+(def sizes
+  {:size-40 {:size             40
+             :icon-size        20
+             :emoji-size       20
+             :emoji-padding    10
+             :padding-vertical 9
+             :padding-left     {:default 16 :icon 12}
+             :padding-right    12
+             :border-radius    12
+             :text-size        :paragraph-1}
+   :size-32 {:size             32
+             :icon-size        20
+             :emoji-size       20
+             :emoji-padding    6
+             :padding-vertical 5
+             :padding-left     {:default 12 :icon 8}
+             :padding-right    8
+             :border-radius    10
+             :text-size        :paragraph-1}
+   :size-24 {:size             24
+             :icon-size        12
+             :emoji-size       12
+             :emoji-padding    6
+             :padding-vertical 3
+             :padding-left     {:default 8 :icon 8}
+             :padding-right    8
+             :border-radius    8
+             :text-size        :paragraph-2}})
+
+(defn- custom-color-type
+  [theme customization-color active?]
+  {:left-icon-color    colors/white-opa-70
+   :right-icon-color   colors/white-opa-20
+   :right-icon-color-2 colors/white
+   :label-color        colors/white
+   :background-color   (if active?
```

Comment:
@smohamedjavid - I think with this we should be using a similar approach described as the on press for the button component - among other custom color pressables.

See here: https://www.notion.so/Colors-mechanism-d4060bc87cfb4c268d19a77f0711d3a1

albeit this pr looks good and perhaps it's better to do as a follow up afterwards? 🤔

Reply:
basically you use an overlay to handle the custom color as this allows for hex strings to be passed (network colors/community colors) and leaves the color resolver to be using `theme` in place of these 50/60 values

Reply:
Thanks for the link, @J-Son89.

I can add the overlay in this PR, as sometimes follow-up is lost in the process. 🥲 👍

Reply:
great! an example use is here in the button https://github.com/status-im/status-mobile/blob/develop/src/quo2/components/buttons/button/view.cljs#L80-L84

Reply:
Thank you! I have added the overlay. 😄

Reply:
@smohamedjavid thanks for addressing the comment! Just a small request for the future: if some changes are made after PR is already in process of testing please ping up QAs about this. In such cases we need to re-start testing on new builds that include new changes and re-run out e2e. Otherwise we might miss some bugs which might appear as a result of new untested changes. Thank you!

Reply:
Noted @pavloburykh. I will ensure to re-run the E2E tests, if any changes are made during testing.
Apologies for the inconvenience.

Reply:
> Noted @pavloburykh. I will ensure to re-run the E2E tests, if any changes are made during testing. Apologies for the inconvenience.

no worries @smohamedjavid Thank you for your work. Additional thanks for detailed PR description. It helps a lot.